libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -4255,10 +4258,10 @@ void GCS_MAVLINK::send_sys_status()
 
     const AP_BattMonitor &battery = AP::battery();
     float battery_current;
-    int8_t battery_remaining;
+    int8_t battery_remaining = 1;
```

Comment:
Could you put a comment here why you are defaulting to 1 instead of 0 ?

Reply:
sorry, typo, I missed a "-"